static/app/views/performance/transactionSummary/transactionTags/tagsDisplay.tsx

```diff
@@ -33,6 +105,33 @@ const TagsDisplay = (props: Props) => {
     projects,
     eventView
   );
+
+  const handleCursor = (cursor: string, pathname: string, query: Query) => {
+    browserHistory.push({
+      pathname,
+      query: {...query, [TAG_PAGE_TABLE_CURSOR]: cursor},
+    });
+  };
+
+  const cursor = decodeScalar(location.query?.[TAG_PAGE_TABLE_CURSOR]);
+
+  const tagSort = getTagSortForTagsPage(location);
+
+  const tagSorts = fromSorts(tagSort);
+
+  eventView.fields = TAGS_TABLE_COLUMN_ORDER;
```

Comment:
Are there any unintended side effects from mutating the `eventView` in the props? Should this create a new event view? Perhaps using `withColumns(...)`.

Reply:
I'm just going to clone for the whole tagsDisplay :+1: